presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-hdp3/hadoop-entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -euo pipefail
+
+fail() {
+  echo "$(basename "$0"): $*" >&2
+  exit 1
+}
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
This should be sufficient:

```suggestion
DIR="${BASH_SOURCE%/*}"
```

(like we did in `apply-site-xml-override.sh`)